activerecord/lib/active_record/store.rb

```diff
@@ -241,12 +253,16 @@ def self.prepare(object, attribute)
       end
```

Comment:
Unrelated, but maybe change the `send` ☝🏻 to `public_send` while we're nearby?

Reply:
Done, and I refactored all this a bit further.